apps/backend/src/routers/index.ts

```diff
@@ -5,13 +5,15 @@ import courseRouter from './course';
 import websocRouter from './websoc';
 import gradesRouter from './grades';
 import enrollHistRouter from './enrollHist';
-import searchRouter from "./search";
+import searchRouter from './search';
+import sessionRouter from './session';
 
 const appRouter = router({
     course: courseRouter,
     enrollHist: enrollHistRouter,
     grades: gradesRouter,
     search: searchRouter,
+    session: sessionRouter,
```

Comment:
Instead of having a `sessionRouter` and a `usersRouter`, there should be an `authRouther` and a `userData` router.